api/v1alpha4/v1alpha4_test.go

```diff
@@ -4,10 +4,9 @@ import (
 	"testing"
 
 	"github.com/nutanix-cloud-native/prism-go-client/environment/credentials"
-
 	"github.com/nutanix-cloud-native/prism-go-client/utils"
 	"github.com/stretchr/testify/assert"
-	"sigs.k8s.io/cluster-api/api/v1alpha4"
+	"sigs.k8s.io/cluster-api/api/v1alpha4" //nolint:staticcheck // we know v1alpha4 is deprecated: release-v1.3 branch is not an active development branch
```

Comment:
instead of saying "we know v1alpha4 is deprecated:" => "Ignoring warning due to inactive dev branch". more mellow stmt